### ISSUE TITLE
windows 10 16257 Nvda doesn't say a notification in Microsoft edge

### ISSUE THREAD

**Issue by gdata1** [nvaccess/nvda#7499]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
> Example:
>
> 1. Open Chrome
> 2. Browse to www.google.com
> 3. Type "Hello"
> 4. Notice an error sound when enter is pressed.

> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
> Example: next-14027,c80e529f

NVDA Installed or portable:

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result
Hello. Starting with Windows 10 16257 NVDA does not read notifications in edge.
Try the following: Start downloading any file.
Expected. Nvda automatically reads the download request and progress. Actual. The query must be searched for yourself.

**Comment by josephsl** [comment]
Hi, I can confirm that live region change event isn’t fired, evidenced by the fact that Narrator too can’t read notifications. I’ll go ahead and send an emergency feedback to MS. Thanks for bringing this up.

**Comment by josephsl** [comment]
Feedback: https://aka.ms/kr13aq

**Comment by ehollig** [comment]
@josephsl, is this related to, or a duplicate of #5807

**Comment by josephsl** [comment]
Hi,

Closing, as this was resolved in a later build.

Thanks.
